requirements.txt

```diff
@@ -2,3 +2,4 @@
 pillow~=10.0.0
 pygame~=2.5.1
 pre-commit~=3.4.0
+scipy~=1.11.2
```

Comment:
Will also want to add numpy as well.

Reply:
I find `numpy` gets installed with `scipy` by default